src/main.c

```diff
@@ -420,12 +408,102 @@ unsigned int ui_approval_prepro(const bagl_element_t *element)
     return display;
 }
 
-unsigned int ui_approval_nanos_button(unsigned int button_mask,
-                                      unsigned int button_mask_counter);
+const bagl_element_t ui_multiple_action_tx_approval_nanos[] = {
```

Comment:
UI elements for `STREAM_CONFIRM_PROCESSING` intermediary state. Basically, notifies user about number of actions that are needed to be approved.